dev-db/etcd/files/etcd.service

```diff
@@ -6,6 +6,8 @@ User=etcd
 PermissionsStartOnly=true
 Environment=ETCD_DATA_DIR=/var/lib/etcd
 Environment=ETCD_NAME=%m
-ExecStart=/usr/bin/etcd 
+ExecStart=/usr/bin/etcd
 Restart=always
 RestartSec=10s
+LimitNOFILE=1048576
```

Comment:
> Does it need to be that high?

It's the same value we're using elsewhere, so I'm just copying that - 

https://github.com/coreos/coreos-overlay/blob/342ea64a4d1bfdc7aeb842c3583b9399ec9c16ff/app-emulation/docker/files/docker.service#L11

https://github.com/coreos/coreos-overlay/blob/342ea64a4d1bfdc7aeb842c3583b9399ec9c16ff/app-emulation/docker/files/early-docker.service#L10

https://github.com/coreos/coreos-overlay/blob/df3eb13e36eda7cc2c680cc714ac5c14f08d8268/app-admin/flannel/files/flanneld.service#L16